tests/unit/postTestBuildComment.ts

```diff
@@ -70,7 +74,7 @@ describe('Post test build comments action tests', () => {
         when(core.getInput).calledWith('IOS_LINK').mockReturnValue('https://expensify.app/IOS_LINK');
         when(core.getInput).calledWith('WEB_LINK').mockReturnValue('https://expensify.app/WEB_LINK');
         when(core.getInput).calledWith('DESKTOP_LINK').mockReturnValue('https://expensify.app/DESKTOP_LINK');
-        createCommentMock.mockResolvedValue(true);
+        createCommentMock.mockResolvedValue({} as RestEndpointMethodTypes['issues']['createComment']['response']);
```

Comment:
Extract `RestEndpointMethodTypes['issues']['createComment']['response']` to a separate type